### ISSUE TITLE
[Malpedia] 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by V1D1AN** [OpenCTI-Platform/connectors#126]
## Description

I have an error with the malpedia connector

## Environment

1. OS (where OpenCTI server runs): CentOs
2. OpenCTI version: OpenCTI 3.2.1
3. OpenCTI client: frontend
4. Other environment details: Run with docker swarm

## Reproducible Steps
git clone .https://github.com/OpenCTI-Platform/connectors.git
cd connectors/malpedia
docker build -t connector-malpedia:3.2.1 .

I have update my docker compose with the good image, the good api key, and i run the command:
env $(cat .env | grep ^[A-Z] | xargs) docker stack deploy --compose-file docker-compose.yml opencti

## Expected Output

```
INFO:root:Listing stix_relations with {type: indicates, from_id: d695c308-c2cb-41cf-b6a4-f073b458c042, to_id: 284b20b7-dc65-44e7-b665-7df2e03ce7a6}
INFO:root:Creating stix_relation {indicator: d695c308-c2cb-41cf-b6a4-f073b458c042, characterize: 284b20b7-dc65-44e7-b665-7df2e03ce7a6}.
INFO:root:{"errors":[{"message":"An unknown error has occurred!  Please try again later.","name":"Unknown","time_thrown":"2020-05-04T09:07:50.613Z","data":{"type":"technical","level":"error"}}]}

ERROR:root:'NoneType' object is not subscriptable
```

this is the end of my docker-compose:
```
      - MALPEDIA_INTERVAL_SEC=86400 # Run once every day
      - CONNECTOR_UPDATE_EXISTING_DATA=true
      - MALPEDIA_IMPORT_ACTORS=false
      - MALPEDIA_IMPORT_YARA=false

```

**Comment by jlequen** [comment]
I have the same problem

**Comment by richard-julien** [comment]
Can you check the log on opencti side? Thanks

**Comment by V1D1AN** [comment]
it works for the moment with Import yara to false.
i will check with import yara to True.
 
P.S. : Merci pour votre taf, un super outil ;)

**Comment by rhaist** [comment]
I fixed a couple of bugs yesterday. Please test with the current :rolling container or the  code on master.

**Comment by V1D1AN** [comment]
I have an error with yara import to yes
```
| push                edx\n            //   8d9e94000000         | lea                 ebx, [esi + 0x94]\n            //   ff15????????         |                     \n            //   57                   | push                edi\n\n    condition:\n        7 of them and filesize < 188416\n}"], "operator": "match"}].
INFO:root:Listing stix_relations with {type: indicates, from_id: cef4b7dc-a30f-41c4-8cc7-f43b71deb54a, to_id: 46de8549-16af-4173-80f2-2cd4ce3eb142}
INFO:root:Creating stix_relation {indicator: cef4b7dc-a30f-41c4-8cc7-f43b71deb54a, characterize: 46de8549-16af-4173-80f2-2cd4ce3eb142}.
INFO:root:{"errors":[{"message":"An unknown error has occurred!  Please try again later.","name":"Unknown","time_thrown":"2020-05-08T11:15:18.797Z","data":{"type":"technical","level":"error"}}]}

ERROR:root:'NoneType' object is not subscriptable
```

**Comment by rhaist** [comment]
Hey @V1D1AN could you please test with the latest rolling release?
There where improvements from @SamuelHassine on the python client and I also improved the connector.

**Comment by V1D1AN** [comment]
Hi,

With the last version, malpedia don't work.

```
INFO:root:Reading Marking-Definition {marking-definition--f88d31f6-486f-44da-b317-01333bde0b82}.
INFO:root:Listing Stix-Domain-Entities with filters [{"key": "name", "values": ["Malpedia"]}].
INFO:root:{"errors":[{"message":"Cannot query field \"contact_information\" on type \"Identity\".","locations":[{"line":13,"column":17}],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"}}]}

```

I have delete the url of malpedia in the docker-compose..
Now, even if i put "true" or "false" in import YARA and import actor,

Malpedia don't work

**Comment by SamuelHassine** [comment]
Yes @V1D1AN, this is a new field that is for the moment only in the master branch. Please wait for the final release of the 3.3.0.

**Comment by SamuelHassine** [comment]
This problem should now be solved with the latest release 3.3.1. Please feel free to re-open if you have any problems.
